pmlc/conversion/gpu/gpu_to_vulkan.cc

```diff
@@ -448,11 +449,16 @@ void ConvertGpuLaunchFuncToVulkanCalls::convertGpuLaunchFunc(
     return signalPassFailure();
   }
 
+  // TODO: select subgroup size based on the previous passes and some special
```

Comment:
So technically we cannot set subgroupSize for non-subgroup kernels to 8 if the number of threads is not divisible by 8.  I think we need to grab the attribute on the kernel launch and pass that down.  non-subgrouped kernels should either have no attribute or have an attribute set to 1 depending on lowering stage.